packages/wrangler/src/dev/hotkeys.ts

```diff
@@ -30,20 +27,6 @@ export default function registerDevHotKeys(
 				);
 			},
 		},
-		{
-			keys: ["l"],
-			disabled: () => args.forceLocal ?? false,
-			label: () =>
-				`turn ${devEnv.config.latestConfig?.dev?.remote ? "on" : "off"} local mode`,
-			handler: async () => {
-				await devEnv.config.patch({
-					dev: {
-						...devEnv.config.latestConfig?.dev,
-						remote: !devEnv.config.latestConfig?.dev?.remote,
-					},
-				});
-			},
-		},
```

Comment:
Is this not a breaking change?

Reply:
Ah yeah, I suppose it is. I'll re-add it

Reply:
I would be up for hiding it from display but still supporting the key-press.